obelisk_ws/src/obelisk_ros/config/leap_py.yaml

```diff
@@ -0,0 +1,94 @@
+config: leap
+onboard:
+  control:
+    pkg: obelisk_control_py
+    executable: leap_controller
+    # callback_groups:
+    publishers:
+      - ros_parameter: pub_ctrl_setting
+        topic: /obelisk/leap/ctrl
+        msg_type: PositionSetpoint
+        key: "asdf"
```

Comment:
probably remove this "asdf" key that i accidentally committed for testing purposes earlier